plugins/Referrers/Columns/Base.php

```diff
@@ -111,6 +111,10 @@ protected function getReferrerInformation($referrerUrl, $currentUrl, $idSite, Re
             $urlsByHost = $this->getCachedUrlsByHostAndIdSite();
 
             $directEntry = new SiteUrls();
+            if ($this->isSiteHasNoUrls($idSite)) {
```

Comment:
Thought it might be better to avoid any possible attacks, since a user could specify any domain they want and maybe mess w/ the referrer URL reports. But after thinking about it I don't think it's possible.